dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -85,7 +86,7 @@ class ScriptSeedTest < ActiveSupport::TestCase
       # this is slower for most individual Scripts, but there could be a savings when seeding multiple Scripts.
       # For now, leaving this as a potential future optimization, since it seems to be reasonably fast as is.
       # The game queries can probably be avoided with a little work, though they only apply for Blockly levels.
-      assert_queries(94) do
+      assert_queries(95) do
```

Comment:
Should the comment above be adjusted for the extra query?

Reply:
@davidsbailey This is a result of the search for all the programming environments right? Did you add a comment for standards or should I add one for each?

Reply:
> @davidsbailey This is a result of the search for all the programming environments right?

Yes

> Did you add a comment for standards or should I add one for each?

I did not. it would be great if you could add it here. I'm not super confident the rest of the counts in this comment are accurate, but I suppose it could still help to add accurate comments when we do know where the queries are coming from.

Reply:
Yeah I'm not sure if this comment is accurate, but I thought it might be good to at least update for the extra query in this PR. I wouldn't spend a lot of time trying to account for every query right now